modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-report-web/src/main/resources/META-INF/resources/js/components/card-shortcut/CardShortcut.es.js

```diff
@@ -29,21 +29,31 @@ export default ({fields}) => {
 		}
 	});
 
+	const setSearchParamsWithoutPageReload = (param, index) => {
+		const url = new URL(location.toString());
+		url.searchParams.set(param, index);
+		window.history.replaceState({path: url.toString()}, '', url.toString());
```

Comment:
@brunofariasdeo, can we add blank spaces after `{` and before `}`? Did the SF check complain about it? In addition, do we need to replace the state of browser's history to make it work?